packages/browser/package.json

```diff
@@ -60,6 +60,8 @@
     "build:bundle:watch": "rollup --config --watch",
     "build:dist": "tsc -p tsconfig.build.json",
     "build:dist:watch": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
+    "build:es5": "tsc -p tsconfig.build.json",
+    "build:es5:watch": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
```

Comment:
Whichever you prefer. I just want to make sure we don't duplicate the same commands, as it can be easy to miss the addition/removal of the flags if we ever decide to modify the command.

Reply:
Good point. Done.